src/Blacklist.hs

```diff
@@ -114,19 +114,18 @@ nameList =
 
 contentList :: Blacklist
 contentList =
-  [ infixOf "DO NOT EDIT" "Derivation file says not to edit it."
-  , infixOf "Do not edit!" "Derivation file says not to edit it."
+  [ infixOf "nixpkgs-update: no auto update" "Derivation file opts-out of auto-updates"
+  , infixOf "DO NOT EDIT" "Derivation file says not to edit it"
+  , infixOf "Do not edit!" "Derivation file says not to edit it"
     -- Skip packages that have special builders
-  , infixOf "buildGoPackage" "Derivation contains buildGoPackage."
-  , infixOf "buildRustCrate" "Derivation contains buildRustCrate."
-  --, infixOf "buildPythonPackage" "Derivation contains buildPythonPackage." -- FRidh says lets' try python
-  , infixOf "buildRubyGem" "Derivation contains buildRubyGem."
-  , infixOf "bundlerEnv" "Derivation contains bundlerEnv."
-  , infixOf "buildPerlPackage" "Derivation contains buildPerlPackage."
-  , infixOf
-      "nixpkgs-update: no auto update"
-      "Derivation file asks not to auto update it"
-  , infixOf "gnome" "do not update GNOME during a release cycle"
+  , infixOf "buildGoPackage" "Derivation contains buildGoPackage"
+  , infixOf "buildRustCrate" "Derivation contains buildRustCrate"
+  , infixOf "buildRubyGem" "Derivation contains buildRubyGem"
+  , infixOf "bundlerEnv" "Derivation contains bundlerEnv"
+  , infixOf "buildPerlPackage" "Derivation contains buildPerlPackage"
+    -- Specific skips for classes of packages
+  , infixOf "https://downloads.haskell.org/ghc/" "GHC packages are versioned per file"
```

Comment:
This line is the new one.  Basically, if the derivation file contains the string `https://downloads.haskell.org/ghc/`, @r-ryantm will skip the update entirely, like so:

```
2020-03-07T15:21:07 New run of nixpkgs-update
2020-03-07T15:21:07 ghc 8.8.2 -> 8.8.3
2020-03-07T15:21:52 FAIL GHC packages are versioned per file
2020-03-07T15:21:53 nixpkgs-update finished
```

Reply:
Thanks!